namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-core-services-tooling/resources/sonarqube-db.tf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+
+module "pcs_sonarqube_rds" {
+  source                      = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.7"
+  cluster_name                = "${var.cluster_name}"
+  cluster_state_bucket        = "${var.cluster_state_bucket}"
+  team_name                   = "${var.team_name}"
+  business-unit               = "${var.business-unit}"
+  application                 = "${var.application}"
+  is-production               = "${var.is-production}"
+  environment-name            = "${var.environment-name}"
+  infrastructure-support      = "${var.infrastructure-support}"
+  db_allocated_storage        = "50"
+  db_instance_class           = "db.t3.medium"
+  db_engine_version           = "9.6"
+  rds_family                  = "postgres9.6"
+  allow_major_version_upgrade = "true"
+
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = "aws.london"
+  }
+}
+
+resource "kubernetes_secret" "pcs_sonarqube_rds" {
+  metadata {
+    name      = "pcs_sonarqube_rds"
```

Comment:
I think you need to replace the underscores here with hyphens.

This is the pipeline error:

```
Error: kubernetes_secret.pcs_sonarqube_rds: metadata.0.name a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

Reply:
OK, done, thanks